### DISCUSSION TITLE
[Released] Add support for git LFS

### DISCUSSION THREAD

**Discussion by donalffons** [vercel/community#1958] [Idea]
I wasted a lot of time today, figuring out that the Vercel Github integration doesn't support git LFS. All I got was the error message ```unsupported file type: undefined``` when trying to `import` images that have been saved using LFS. That error message was not helpful at all and researching the error lead me down a completely wrong path.

Anyway, please either add  support for git LFS or show a meaningful error (or warning) message instead.

I'm a big fan of your project otherwise. Cheers!

**Comment by janpio** [comment]
I think instead of the actual file that is in Git LFS, it is deploying the "pointer" that is included in the actual Git repo - which is muuuuuch smaller and only includes some information what the actual file would look like. I noticed when trying to read the content of my LFS stored file.

**Comment by amyegan** [comment]
We have [Git LFS](https://vercel.com/docs/concepts/projects/overview#git-large-file-storage-lfs) support now: https://vercel.com/changelog/git-lfs-support 🎉
